### PR TITLE
Update to 1.66

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
@ocefpaf @isuruf The new version fails to build on windows, any idea what's wrong ?

File: recipe/bld.bat [hunk 1]
```diff
@@ -12,21 +12,27 @@ if errorlevel 1 exit 1
     threading=multi ^
     link=static,shared ^
     -j%CPU_COUNT% ^
-    --without-python
+    --without-python -d+2
```
Comment (jakirkham) [inline recipe/bld.bat]: What’s this last bit for?

Comment (jschueller) [reply recipe/bld.bat]: this enable b2 debug output, but it turns out the build was okay, it's just that the install layout changed

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
@jakirkham @ocefpaf anyone knows why the initiatliation of conda on win/py27 fails with "The batch file cannot be found." ?

**Comment by isuruf** [comment]
https://github.com/conda/conda/issues/6527

**Comment by jschueller** [comment]
thanks

**Pull request merged**